src/main/scala/org/citegraph/Application.scala

```diff
@@ -13,18 +20,51 @@ object Application {
     println("USAGE")
     println("\tBuild:\n\t\tsbt package")
     println("\tSubmit as JAR to Spark cluster:\n\t\t$SPARK_HOME/bin/spark-submit <submit_options> \\")
-    println("\t\ttarget/scala-2.13/citegraph_2.13-0.1.jar <hdfs_file>")
-    println("\t\trequires option csv_directory needs to have path to where the data csv files are")
+    println("\t\ttarget/scala-2.13/citegraph_2.13-0.1.jar <hdfs_input_dir> <hdfs_output_dir>")
     println()
   }
 
+  def isValidHdfsUri(uri: String): Boolean = {
+    uri.startsWith("hdfs://")
+  }
+
   def main(args: Array[String]): Unit = {
     printArgs(args)
     if (args.length != 2) {
       printUsage()
       System.exit(1)
     }
 
+    // Validate and fix inputDirectory
+    var inputDirectory: String = args(0)
+    val outputDirectory: String = args(1)
+    if (!isValidHdfsUri(inputDirectory)) {
+      printf("Invalid HDFS input directory: %s\n", inputDirectory)
+      System.exit(1)
+    } else if (!isValidHdfsUri(outputDirectory)) {
+      printf("Invalid HDFS output directory: %s\n", outputDirectory)
+      System.exit(1)
+    }
```

Comment:
These guard statements probably aren't necessary, but I like having them in here.